### PR TITLE
Copy over autogenerate tool from k8s/community

### PR DESCRIPTION
**Pull request by swiftdiaries**
1. Copy over generate tool from k8s/community
2. Copy over and combine wgs into one single YAML
3. Autogen from generate tool

### CODE REVIEW

**Comment by swiftdiaries** [comment]
@animeshsingh @ellistarn 
Could you please ask @deadeyegoodwin to make a PR similar to https://github.com/kubeflow/internal-acls/pull/316 and add themselves to the Kubeflow org?
They are added as a OWNER of the KFServing WG but it looks they're not part of the org.

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,23 @@
+aliases:
+  wg-automl-leads:
```
Comment (jlewi) [inline OWNERS_ALIASES:2]: Do we need the prefix "wg"? Is it redundant with leads?

**Comment by jlewi** [comment]
Thanks for doing this!

File: OWNERS_ALIASES (hunk 1 above)
Comment (jlewi) [reply OWNERS_ALIASES:2]: I checked what Kubernetes does. Lets keep the wg prefix.

File: wgs/wg-automl/charter.md [hunk 2]
```diff
@@ -1,39 +0,0 @@
-# WG AutoML Charter
```
Comment (andreyvelich) [inline wgs/wg-automl/charter.md:1]: Why `charter.md` is deleted?

File: wgs.yaml [hunk 3]
```diff
@@ -0,0 +1,208 @@
+sigs: []
```
Comment (andreyvelich) [inline wgs.yaml:1]: Should we add all information related to wg under `/community/wgs` or we want to follow k8s structure way where we add folders to `/community`

**Comment by swiftdiaries** [comment]
We can probably add him back in subsequent PR after he creates a PR adding himself to the Kubeflow org. I hope that's ok.

File: wg-automl/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+To understand how this file is generated, see https://github.com/kubeflow/community/generator/README.md
+--->
+# AutoML Working Group
+
+WG AutoML is responsible for all aspects of Automated Machine Learning technologies on Kubeflow.
+
+The [charter](charter.md) defines the scope and governance of the AutoML Working Group.
+
+## Meetings
+* Regular Katib Meeting (Asia & Europe friendly): [Wednesdays at 10:00 am UTC (Coordinated Universal Time)]() (monthly - second Wednesday every month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00 am&tz=UTC%20%28Coordinated%20Universal%20Time%29).
```
Comment (andreyvelich) [inline wg-automl/README.md:16]:
```suggestion
* Regular Katib Meeting (Asia & Europe friendly): [Wednesdays at 10:00 am UTC (Coordinated Universal Time)]() (monthly - second Wednesday every month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00am&tz=UTC%20%28Coordinated%20Universal%20Time%29).
```

**Comment by andreyvelich** [comment]
Thank you @swiftdiaries!
I added few comments.

File: wgs/wg-automl/charter.md (hunk 2 above)
Comment (swiftdiaries) [reply wgs/wg-automl/charter.md:1]: The charter is now auto-generated from the `wgs.yaml` and placed into `wg-*/README.md`

File: wg-automl/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+* Andrey Velichkevich (**[@andreyvelich](https://github.com/andreyvelich)**), Cisco
+* Ce Gao (**[@gaocegege](https://github.com/gaocegege)**), Caicloud
+* Johnu George (**[@johnugeorge](https://github.com/johnugeorge)**), Cisco
+
+## Contact
+- Slack: [#wg-automl](https://kubernetes.slack.com/messages/wg-automl)
+- [Mailing list](https://groups.google.com/forum/#!forum/kubeflow-discuss)
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/wg%2Farea/wg-automl)
+- GitHub Teams:
+    - [@kubernetes/wg-automl-bugs](https://github.com/orgs/kubernetes/teams/wg-automl-bugs) - Bug Triage and Troubleshooting
```
Comment (andreyvelich) [inline wg-automl/README.md:32]: Should we change it to `https://github.com/orgs/kubeflow/teams/....` ?

File: wgs.yaml (hunk 3 above)
Comment (swiftdiaries) [reply wgs.yaml:1]: You don't add anything. You ***only*** make changes to the `wgs.yaml` file. PTAL at the `generator/README.md` for more information on how it works

Comment (andreyvelich) [reply wgs.yaml:1]: Yeah, I got it.
I am just thinking do we want to modify generate script to follow steps, introduced here: https://github.com/kubeflow/community/blob/master/wgs/wg-lifecycle.md.

File: wg-automl/README.md (hunk 5 above)
Comment (swiftdiaries) [reply wg-automl/README.md:32]: I have some leftovers from the templates. Let me remove them

File: wg-automl/README.md (hunk 4 above)
Comment (swiftdiaries) [reply wg-automl/README.md:16]: The `wgs.yaml` had a redundant space that was kept as is. Let me regenerate with the change in `wgs.yaml`

**Comment by swiftdiaries** [comment]
Thanks for the review @andreyvelich :)

File: wgs.yaml [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+  meetings:
+  - description: Regular Katib Meeting (Asia & Europe friendly)
+    day: Wednesday
+    time: 10:00am
+    tz: UTC (Coordinated Universal Time)
+    frequency: monthly - second Wednesday every month
+    archive_url: https://docs.google.com/document/d/1MChKfzrKAeFRtYqypFbMXL6ZIc_OgijjkvbqmwRV-64
+  - description: Regular Katib Meeting (US friendly)
+    day: Wednesday
+    time: 4:00 pm
```
Comment (andreyvelich) [inline wgs.yaml:41]: I think you should also remove space from here.
```suggestion
    time: 4:00pm
```

File: wg-automl/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://github.com/kubeflow/community/generator/README.md
+--->
+# AutoML Working Group
+
+WG AutoML is responsible for all aspects of Automated Machine Learning technologies on Kubeflow.
+
+The [charter](charter.md) defines the scope and governance of the AutoML Working Group.
```
Comment (andreyvelich) [inline wg-automl/README.md:13]: Do we need to link to the `charter.md` that doesn't exits?

**Comment by animeshsingh** [comment]
To clarify - what was the reasoning for merging wg yaml files into one? Weren't they better managed individually?

File: wgs/wg-automl/charter.md (hunk 2 above)
Comment (swiftdiaries) [reply wgs/wg-automl/charter.md:1]: I mis-understood the charter part. I'm adding them back under `wg-*/charter.md`

File: wgs.yaml (hunk 3 above)
Comment (swiftdiaries) [reply wgs.yaml:1]: I've edited that part [here](https://github.com/kubeflow/community/blob/618908e73928afb827649dc55a3131b749564820/wgs/wg-lifecycle.md#github) to follow new instructions

File: wg-automl/README.md (hunk 7 above)
Comment (swiftdiaries) [reply wg-automl/README.md:13]: > I mis-understood the charter part. I'm adding them back under wg-*/charter.md
Sorry I just can't seem to concentrate today :( 
Fixing it.

**Comment by swiftdiaries** [comment]
@animeshsingh 
https://github.com/kubeflow/community/issues/398#issuecomment-679205312

File: wg-automl/README.md (hunk 7 above)
Comment (andreyvelich) [reply wg-automl/README.md:13]: It's totally fine, thank you for doing this!

**Review by andreyvelich — APPROVED** [review]
Thanks for doing this @swiftdiaries!
/lgtm

**Comment by swiftdiaries** [comment]
Thanks for the reviews :)

**Pull request merged**